### PR TITLE
Feature/henry/article clensing

### PR DESCRIPTION
**Pull request by achaccha**
I extract 'publish'  same format and remove needless information in 'content'.
Also I fixed before code review #1 .
We'll optimize more after this commit.
Please take a look.

### CODE REVIEW

File: crawler/rss.py [hunk 1]
```diff
@@ -29,26 +31,51 @@ def extract(cls, objs, code, options):
...
\ No newline at end of file
+        return ret
+
+    @classmethod
+    def parse_date(cls, date_string):
+        """
+        Unify date in different format of each news in the
+        same format.
+        """
+        new_date = parser.parse(date_string)
```
Comment (ethkim) [inline crawler/rss.py:71]: How do we deal with date format that can't be parsed? I guess all kinds of weird datetime formats are being used per different providers. We need a way to gracefully break when this fails.

File: crawler/rss.py [hunk 2]
```diff
@@ -29,26 +31,51 @@ def extract(cls, objs, code, options):
...
+        new_date_string = str(new_date) 
+        return new_date_string
+
+    @classmethod
+    def parse_content(cls, content_string):
+        """
+        Remove needless texts from content.
+        """
+        new_content_string = content_string.split("Read more...")[0]
```
Comment (ethkim) [inline crawler/rss.py]: Add TODO to detect "needless text".

**Review by ethkim — APPROVED** [review]
Thanks!

**Comment by ethkim** [comment]
Oh, and write more meaningful title for each PR. "article cleansing" doesn't really explain what this does.

File: crawler/rss.py (hunk 1 above)
Comment (harrydrippin) [reply crawler/rss.py:71]: How about making formatted strings for each newspaper? e.g.` "{Y}-{M}-{D} {H}:{M}:{S} {TZ}"`

File: requirements.txt [hunk 3]
```diff
@@ -2,3 +2,4 @@ Flask==0.10.1
 requests==2.18.4
 beautifulsoup4==4.6.0
 lxml==4.1.1
+python-dateutil
```
Comment (harrydrippin) [inline requirements.txt:5]: Need to specify exact version here.

**Review by harrydrippin — COMMENTED** [review]
Thanks! PTAL

**Review by harrydrippin — APPROVED** [review]

File: crawler/rss.py (hunk 2 above)
Comment (achaccha) [reply crawler/rss.py]: I'll fix it on next PR.

File: requirements.txt (hunk 3 above)
Comment (achaccha) [reply requirements.txt:5]: I'll add a version name with `2.6.1`.

File: crawler/rss.py (hunk 1 above)
Comment (achaccha) [reply crawler/rss.py:71]: I'll take a log of the format that can not be parsed.
I'll store such format and make a new parser corresponding to each.

**Pull request merged**